main.py

```diff
@@ -1,12 +1,353 @@
 import requests
+from video_requests import VideoRequests
+from customer_requests import CustomerRequests
+from rental_requests import RentalRequests
+import json
 
-URL = "http://127.0.0.1:5000"
+URL = "http://127.0.0.1:5000"  #local host
 BACKUP_URL = "https://retro-video-store-api.herokuapp.com"
+#MY_URL = "https://video-retro.herokuapp.com" #api route
+
+video_requests = VideoRequests(url=BACKUP_URL)
+customer_requests = CustomerRequests(url=BACKUP_URL)
+rental_requests = RentalRequests(url=BACKUP_URL)
```

Comment:
If you notice these three classes are doing the same things.  All are basically interacting with the API on behalf of `main.py`.  I suggest instead having one class which handles interactions with the API, that would be a little more concise.